tcpdp v0.5.0 (darwin amd64) raises SIGSEGV when doing -help

I tried tcpdp v0.5.0 in Mac (macOS High Sierra 10.13.6 17G65), but `tcpdp -help` raises SIGSEGV like the following:

```console
$ ./tcpdp_v0.5.0_darwin_amd64/tcpdp -help
Error: unknown shorthand flag: 'e' in -elp
Usage:
  tcpdp [command]

Available Commands:
  config      show currnt config
  help        Help about any command
  probe       probe
  proxy       proxy
  version     print tcpdp version

Flags:
  -h, --help   help for tcpdp

Use "tcpdp [command] --help" for more information about a command.

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x446610e]

goroutine 1 [running]:
github.com/k1LoW/tcpdp/vendor/go.uber.org/zap.(*Logger).check(0x0, 0x5, 0x45b4142, 0x5, 0xc4211d6400)
        /Users/k1low/src/github.com/k1LoW/tcpdp/vendor/go.uber.org/zap/logger.go:264 +0x8e
github.com/k1LoW/tcpdp/vendor/go.uber.org/zap.(*Logger).Fatal(0x0, 0x45b4142, 0x5, 0xc4211d6400, 0x1, 0x1)
        /Users/k1low/src/github.com/k1LoW/tcpdp/vendor/go.uber.org/zap/logger.go:235 +0x44
github.com/k1LoW/tcpdp/cmd.Execute()
        /Users/k1low/src/github.com/k1LoW/tcpdp/cmd/root.go:48 +0x1bf
main.main()
        /Users/k1low/src/github.com/k1LoW/tcpdp/main.go:34 +0x3a
```

#### SPEC (My Mac)

```
  Model Name:	MacBook Pro
  Model Identifier:	MacBookPro14,2
  Processor Name:	Intel Core i7
  Processor Speed:	3.5 GHz
  Number of Processors:	1
  Total Number of Cores:	2
  L2 Cache (per Core):	256 KB
  L3 Cache:	4 MB
  Memory:	16 GB
  Boot ROM Version:	MBP142.0178.B00
  SMC Version (system):	2.44f1
  Serial Number (system):	C02VD0Y9HV2R
  Hardware UUID:	1283D383-75BC-5752-8A85-47C2706C5BF7
```

It seems like it occurs panic when giving shorthand flag except for `-h`:

```console
$ ./tcpdp -a
Error: unknown shorthand flag: 'a' in -a
Usage:
  tcpdp [command]

Available Commands:
  config      show currnt config
  help        Help about any command
  probe       probe
  proxy       proxy
  version     print tcpdp version

Flags:
  -h, --help   help for tcpdp

Use "tcpdp [command] --help" for more information about a command.

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x448e81e]

goroutine 1 [running]:
github.com/k1LoW/tcpdp/vendor/go.uber.org/zap.(*Logger).check(0x0, 0x5, 0x45daad9, 0x5, 0xc000148980)
        /Users/hfm/src/github.com/k1LoW/tcpdp/vendor/go.uber.org/zap/logger.go:264 +0x8e
github.com/k1LoW/tcpdp/vendor/go.uber.org/zap.(*Logger).Fatal(0x0, 0x45daad9, 0x5, 0xc000148980, 0x1, 0x1)
        /Users/hfm/src/github.com/k1LoW/tcpdp/vendor/go.uber.org/zap/logger.go:235 +0x44
github.com/k1LoW/tcpdp/cmd.Execute()
        /Users/hfm/src/github.com/k1LoW/tcpdp/cmd/root.go:48 +0x20d
main.main()
        /Users/hfm/src/github.com/k1LoW/tcpdp/main.go:34 +0x3a
```

--- a/README.md
+++ b/README.md
@@ -4,7 +4,7 @@ tcpdp is TCP dump tool with custom dumper written in Go.
 
 ## Usage
 
-### `tcpdp proxy` : Proxy mode
+### `tcpdp proxy` : TCP proxy server mode
 
 ``` console
 $ tcpdp proxy -l localhost:12345 -r localhost:1234 -d hex # hex.Dump()

--- a/cmd/probe.go
+++ b/cmd/probe.go
@@ -40,8 +40,8 @@ var (
 // probeCmd represents the probe command
 var probeCmd = &cobra.Command{
 	Use:   "probe",
-	Short: "probe",
-	Long:  `probe`,
+	Short: "Probe mode",
+	Long:  "`tcp prove` dump packets like tcpdump",
 	Run: func(cmd *cobra.Command, args []string) {
 		err := viper.ReadInConfig()
 		if err != nil {

--- a/cmd/proxy.go
+++ b/cmd/proxy.go
@@ -41,8 +41,8 @@ var (
 // proxyCmd represents the proxy command
 var proxyCmd = &cobra.Command{
 	Use:   "proxy",
-	Short: "proxy",
-	Long:  `proxy`,
+	Short: "TCP proxy server mode",
+	Long:  "`tcpdp proxy` run TCP proxy server.",
 	Run: func(cmd *cobra.Command, args []string) {
 		err := viper.ReadInConfig()
 		if err != nil {

--- a/cmd/root.go
+++ b/cmd/root.go
@@ -45,7 +45,6 @@ var rootCmd = &cobra.Command{
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
 	if err := rootCmd.Execute(); err != nil {
-		logger.Fatal("error", zap.Error(err))
 		os.Exit(1)
 	}
 }

--- a/cmd/root.go
+++ b/cmd/root.go
@@ -37,8 +37,8 @@ var (
 // rootCmd represents the base command when called without any subcommands
 var rootCmd = &cobra.Command{
 	Use:   "tcpdp",
-	Short: "tcpdp",
-	Long:  `tcpdp`,
+	Short: "tcpdp is TCP dump tool with custom dumper written in Go.",
+	Long:  `tcpdp is TCP dump tool with custom dumper written in Go.`,
 }
 
 // Execute adds all child commands to the root command and sets flags appropriately.

